lib/platform/bitbucket-server/index.js

```diff
@@ -277,7 +277,7 @@ async function deleteBranch(branchName, closePr = false) {
   if (closePr) {
     // getBranchPr
     const pr = await getBranchPr(branchName);
-    if (pr) {
+    if (pr && pr.state == 'open') {
```

Comment:
getBranchPr should not return a PR unless its state is open

Reply:
Ok, I will remove that.